addons/mail/security/ir.model.access.csv

```diff
@@ -18,6 +18,10 @@ access_discuss_channel_member_public,discuss.channel.member.public,model_discuss
 access_discuss_channel_member_portal,discuss.channel.member.portal,model_discuss_channel_member,base.group_portal,1,1,1,1
 access_discuss_channel_member_user,discuss.channel.member.user,model_discuss_channel_member,base.group_user,1,1,1,1
 access_discuss_channel_rtc_session_system,discuss.channel.rtc.session.system,model_discuss_channel_rtc_session,base.group_system,1,1,1,1
+access_discuss_call_history_system,discuss.call.history.system,model_discuss_call_history,base.group_system,1,0,0,0

```

Comment:
Why would system have reduced rights? This is not a typical pattern.
But if we keep it like this system has user as implied so the line is not necessary.